Update README.md (#51)
pyladies-bratislava/website: README.md

1. Install [node.js](https://nodejs.org/en/download/).

> 2. In website directory create a new file called .env with this inside:

Reviewer: ```suggestion
2. In the website directory create a new file called .env with:
```
